editor/plugins/script_text_editor.cpp

```diff
@@ -1955,6 +1956,28 @@ static String _get_dropped_resource_line(const Ref<Resource> &p_resource, bool p
 	return vformat("const %s = preload(%s)", variable_name, _quote_drop_data(path));
 }
 
+String ScriptTextEditor::_get_dropped_resource_ref_line(const Ref<Resource> &p_resource) {
```

Comment:
this method is a member function while `_get_dropped_resource_str_line` is a static function not related to `ScriptTextEditor`

come to think of it is already shady `_get_dropped_resource_str_line` exists here, because this class is not GdScript specific, and these functions are 🤔